guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -1,5 +1,5 @@
-[id="satellite-ports-and-firewalls-requirements_{context}"]
-= Ports and Firewalls Requirements
+[id="ports-and-firewall-requirements_{context}"]
+= Ports and Firewall Requirements
```

Comment:
Please keep the old id along with the new one, we need to do this at least for one release not to break our CI.